<launch>

	<arg name="image_topic" default="/camera/color/image_raw" />

	<node pkg="yolact_ros" type="yolact_ros_node.py" name="yolact_ros" output="screen">

		<param name="image_topic" value="$(arg image_topic)" />

	</node>

</launch>
